Formula/pcl.rb

```diff
@@ -67,6 +59,10 @@ def install
       "-DBUILD_apps_modeler:BOOL=OFF"
     end
 
+    if OS.mac? && ((MacOS.version == :monterey) || (MacOS.version == :ventura)) && Hardware::CPU.arm?
```

Comment:
why not on Big Sur?

Reply:
@SMillerDev Could you add the long-timeout label, please? (unless I am mistaken and the GitHub Actions weren't cancelled because of timeouts)